File: MSOE_2016.c
#include <stdio.h>

int main() {
    int n;
    int count;

    printf("Enter an int between 1 and 10000: ");
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        if (i % 2 != 0 && i % 3 != 0 && i % 5 != 0) count++;
    }

    printf("%d", count);
    printf("\n");
}

File: msoe2016.py
def main():
    n = int(input("Enter a number 1 to 10000: "))
    count = 0
    
    for i in range(1, n):
        if n % i == 0: count+=1
    count+=1
    print(count)



if __name__ == "__main__":
    main()